website/client/src/components/inventory/equipment/equipGearModal.vue

```diff
@@ -94,22 +88,20 @@
       padding: 2rem 1.5rem;
     }
 
-    .close {
+    .dialog-close {
```

Comment:
in `website/client/assets/scss/icon.scss` we have a class `.close-icon` which is used in several modals, I think we should find a way to abstract it as well (it includes the positioning in the top right of the modals) and use only one class for all modals

Reply:
move `.close-icon` class rules to closeIcon.vue - or - only use .close-icon class here ? 

or did I miss a 3rd option ?

Reply:
I think here it's a bit of a problem because sometimes the close icon is used outside of modals so it need a different positioning. I guess we can adda prop `is-modal-close` which when set to true uses the css positioning for modals?

Reply:
should this use the new component? Also do we want to remove `close-icon` from icons.scss?

Reply:
the newest code uses the new component - if I remove close-icon many other dialogs will probably break, so I guess this will have to wait once all other places use the new component instead